charts/jx-app-jacoco/Makefile

```diff
@@ -38,10 +38,10 @@ else ifeq ($(OS),Linux)
 	sed -i -e "s/version:.*/version: $(VERSION)/" Chart.yaml
 	sed -i -e "s/tag: .*/tag: $(VERSION)/" values.yaml
 else
-	echo "platfrom $(OS) not supported to tag with"
+	echo "platform $(OS) not supported to tag with"
 	exit -1
 endif
 	git add --all
-	git commit -m "release $(VERSION)" --allow-empty # if first release then no verion update is performed
+	git commit -m "release $(VERSION)" --allow-empty # if first release then no version update is performed
 	git tag -fa v$(VERSION) -m "Release version $(VERSION)"
```

Comment:
and here you tag this commit, but the commit itself gets never pushed. This throws the whole changelog generation process, not speaking of never being able to checkout a tag to rebuild a release.